pkg/core/storage.go

```diff
@@ -1,9 +1,9 @@
 package core
 
-type Storage interface {
+type IStorage interface {
```

Comment:
Like this interface will push inside a storage, the correct name can be `Pusher`
```suggestion
type Pusher interface {
```

Reply:
Fixed